app.js

```diff
@@ -224,6 +225,11 @@ app.put('/unpublish',
         middleware.validateRequest(["description", "dateUpdated", "public"]),
         routes.unpublish);
 
+app.get('/remix/:projectId',
+        middleware.setUserIfTokenExists,
+        middleware.publishLogin(true),
```

Comment:
So if you aren't logged into thimble, you still can remix a project. It will just drop the entire project into the editor but you wont be able to persist it!